corehq/feature_previews.py

```diff
@@ -130,3 +130,12 @@ def enable_callcenter(domain_name, checked):
     ),
     privilege=LOOKUP_TABLES,
 )
+
+
+CUSTOM_ICON_BADGES = FeaturePreview(
```

Comment:
Feature previews are being deprecated; this should be an add-on instead: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/add_ons.py